### DISCUSSION TITLE
Blur flag seems not works

### DISCUSSION THREAD

**Discussion by mauroreggio** [pglombardo/PasswordPusher#2328] [Q&A] [answered]
Hi all. I try disable blur in text retrival ... but seems blur can't be disable. Anyone see this too? I run in docker and try use .env file with BLUR options and map config/settings.yml in one local file to, setting "enable_blur: false" to .. but always blur stay enabled. Thanks.

**Comment by pglombardo** [comment]
Hi @mauroreggio,

The quickest way to check is to output the settings value from the application.  Could you use [these instructions](https://docs.pwpush.com/docs/application-console/) to open an application console and then run `Settings.pw.enable_blur` and tell me what the output is?

Also:

  * Which version are you running?
  * Could you post your related configs?  env vars and yml entries?

Thanks

**Comment by mauroreggio** [comment]
Hi Giacomo, buon pomeriggio (bel lavoro, grazie). Thanks for your help.
I paste here what you suggest me:

61801fafe6c1:/opt/PasswordPusher$ bin/pwpush console
Password Pusher Version: 1.41.11
Loading production environment (Rails 7.1.3.4)
irb(main):001> Settings.pw.enable_blur
=> true
irb(main):002>

This means that the setting is ok ... maybe i understand bad what is "blur"? ... i only desire that when the other side receive the link not have to click on "blurred text" for watch at the content.

In my mapped settings.yml file i only change blur option and remove all language that i not need.

  enable_blur: false
  blur_default: false <<<<--- try this to for what i read in file comments  

The other that i say, but blur not can disable before to, is that i modify new.html.erb because i desire have a "minimal" starting page with only text sharing fields (remove password generator that i not need). I arrive at this workaround reading other Q&A not about this args (hope make right).

[dot_env.txt](https://github.com/user-attachments/files/16118975/dot_env.txt)
[settings.yml.txt](https://github.com/user-attachments/files/16118985/settings.yml.txt)
[new.html.erb.txt](https://github.com/user-attachments/files/16118988/new.html.erb.txt)

**Reply by pglombardo** [reply, marked as answer]
Not sure why the `settings.yml` setting didn't work but as for the environment variable, use `PWP__PW__ENABLE_BLUR=false`.  PWP__PW__BLUR was a typo in the settings.yml file - apologies.

**Reply by mauroreggio** [reply]
Yes, this works!!! put PWP__PW__ENABLE_BLUR=false in .env file is the trick.
Thanks Giacomo.
Mauro.

P.s. not know why not works in settings.yml ... i'm sure that mine mapped file is readed bacause i not have all the language that i remove ... so something is readed from mapped file.

**Reply by pglombardo** [reply]
Excellent!  I'm glad its working now.  I'm working on moving many of those settings inside the app as "policies" so hopefully this will be less pain for users.  In progress/coming soon.

![Screenshot 2024-07-07 at 14 32 20](https://github.com/pglombardo/PasswordPusher/assets/395132/f2ae752c-3c55-4a99-b33e-682ae635fbaa)

If you have any other issues, let me know Mauro!
